### DISCUSSION TITLE
Invalid installation. No user with super admin role found!

### DISCUSSION THREAD

**Discussion by ApexDigitalRO** [givanz/Vvveb#211] [General]
Please excuse me, as I am a complete noob when it comes to VVVeb. 

I followed the installation guide for CPANEL. I wanted to test VVVeb on a shared hosting setup. 

I've created a DB, and a added user with all permissions to it. 

So far, so good. However, when I put in the connection details for the DB (incl. localhost instead of 127.0.0.1, and also tried with the local IP), I am taken to the next step with the following error: 

`Invalid installation. No user with super admin role found!`

I can set a language, create an admin just fine, but then I am unable to log in at all. 

I checked the database and it's completely empty. 0B. I tried to test by resetting my password from the login screen, and I am getting a bunch of errors such as: 

`Warning: Undefined array key "email" in ----/admin/controller/user/reset.php on line 76`

I am running this on a subdomain, if it makes any difference. 

What am I doing wrong? 

I am sure that the DB connection details are correct.

**Comment by givanz** [comment]
Hi

Can you please check if tables are created? 

If tables are created then probably insert fails and the default admin account can't be created, the insert sql for admin is [install/sql/insert/admin/admin.sql](https://github.com/givanz/Vvveb/blob/master/install/sql/insert/admin/admin.sql)

If tables are not created please try to create tables using cli or phpmyadmin using the same credentials to rule out any permissions issue.

The sql for admin table is [install/sql/mysqli/schema/admin/admin.sql](https://github.com/givanz/Vvveb/blob/master/install/sql/mysqli/schema/admin/admin.sql)

What version of Mysql/MariaDb are you using?

**Comment by ApexDigitalRO** [comment]
Hi! Thank you so much for your help. After so much troubleshooting, I found out the issue. It was serverside, it seems like mod security prevented the DB from being populated during installation, for some reason. I disabled it and everything seems to be working correctly now.
